### PR TITLE
[WIP] Move the default Identity UI to use Bootstrap 4

### PR DESCRIPTION
**Pull request by javiercn**
# Index page
indexbootstrap4_fullwidth
# Manage account full size
managebootstrap4_fullwidth
# Manage account mid size
managebootstrap4_halfwidth
# Manage account iPhone X size
managebootstrap4_iphonex

### CODE REVIEW

File: src/UI/Areas/Identity/Pages/_Layout.cshtml [hunk 1]
```diff
@@ -47,37 +43,46 @@
...
+        </nav>
+    </header>
+
+    <div class="container-fluid">
+        @{
+            var cookieConsent = Engine.FindView(ViewContext, "_CookieConsentPartial", isMainPage: false);
+        }
+        @if (cookieConsent.Success)
+        {
+            await Html.RenderPartialAsync("_CookieConsentPartial");
```
Comment (pranavkm) [inline src/UI/Areas/Identity/Pages/_Layout.cshtml]: `<partial name="_CookieConsentPartial" optional />`

File: test/WebSites/Identity.DefaultUI.WebSite/wwwroot/Index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿<!DOCTYPE html>
```
Comment (pranavkm) [inline test/WebSites/Identity.DefaultUI.WebSite/wwwroot/Index.html]: Intentional?

File: test/WebSites/Identity.DefaultUI.WebSite/Pages/_Layout.cshtml [hunk 3]
```diff
@@ -10,62 +10,61 @@
...
-                </button>
-                <a asp-page="/Index" class="navbar-brand">Identity.DefaultUI.WebSite</a>
-            </div>
-            <div class="navbar-collapse collapse">
-                <ul class="nav navbar-nav">
-                    <li><a asp-page="/Index">Home</a></li>
-                    <li><a asp-page="/About">About</a></li>
-                    <li><a asp-page="/Contact">Contact</a></li>
-                </ul>
+    <header>
```
Comment (javiercn) [inline test/WebSites/Identity.DefaultUI.WebSite/Pages/_Layout.cshtml]: @ryanbrandenburg Grab the navbar from here

File: test/WebSites/Identity.DefaultUI.WebSite/Pages/Shared/_LoginPartial.cshtml [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
-}
-else
-{
-    <ul class="nav navbar-nav navbar-right">
-        <li><a id="register" asp-area="Identity" asp-page="/Account/Register">Register</a></li>
-        <li><a id="login" asp-area="Identity" asp-page="/Account/Login">Login</a></li>
-    </ul>
-}
\ No newline at end of file
+<ul class="navbar-nav">
```
Comment (javiercn) [inline test/WebSites/Identity.DefaultUI.WebSite/Pages/Shared/_LoginPartial.cshtml]: @ryanbrandenburg This is the login partial markup

**Comment by javiercn** [comment]
https://github.com/aspnet/Identity/commit/4b2dc59979ac57e69f425c2f574b12a6625fe959

**Pull request closed without merging**